core/storage/hamt/hamt.hpp

```diff
@@ -62,67 +67,11 @@ namespace fc::storage::hamt {
     using Item = boost::variant<CID, Ptr, Leaf>;
 
     std::map<size_t, Item> items;
+    boost::optional<bool> v3;
```

Comment:
May be add `version enum` in case we have v4 and etc. Or leave it is a problem of future.